vaadin-portlet-integration-tests/portlet30/pom.xml

```diff
@@ -41,7 +41,7 @@
         <dependency>
             <groupId>com.vaadin</groupId>
             <artifactId>vaadin-button-testbench</artifactId>
-            <version>2.0-SNAPSHOT</version>
+            <version>${vaadin.version}</version>
```

Comment:
In other pom files we use `component.version`. Should we do the same here?

Reply:
Yes, I replaced vaadin version with component version where is it refer to component dependency.